collection/collection.go

```diff
@@ -97,3 +97,84 @@ func Equal[T comparable](s1 []T, s2 []T) bool {
 
 	return true
 }
+
+// ContainsKeys checks whether all items in the slice are in the map
+func ContainsKeys[M ~map[K]V, K comparable, V any](m M, ks []K) bool {
```

Comment:
Can we rename all `Containsxxx` to `Containxxx` including the current `Contains`?

Reply:
>JAVA also have the same naming/
https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/List.html#containsAll(java.util.Collection)

Oh, I never realize about this inconsistency: `containsAll`& `equals` have `s` but others (`addAll`, `removeAll`, `replaceAll`, ...) don't. Do you know why?

>I think it's OK to use Contains bc the subject array and map is a singular

Yeah, I think it's ok to use too. But if we use the same logic then shouldn't a lot of other functions/methods should be in the same form as well?

Go also has `s` for `Contains` for some reason: https://pkg.go.dev/golang.org/x/exp/slices#Contains

Reply:
Ah, your explanation makes sense to me.
I'm ok to change `Equal` to `Equals` too as it fits your thinking above.
Seems like Go team missed it 😂 https://pkg.go.dev/golang.org/x/exp/slices#Equal